streaming/src/protobuf/streaming.proto

```diff
@@ -11,6 +11,12 @@ enum OperatorType {
   SINK = 3;
 }
 
+enum FlowControlType {
+  UNKNOWN_FLOW_CONTROL_TYPE = 0;
```

Comment:
where is this used?

Reply:
The first of enum list is default value in protobuf generating.